src/common/options/mds.yaml.in

```diff
@@ -347,22 +347,22 @@ options:
 - name: mds_session_cap_acquisition_throttle
   type: uint
   level: advanced
-  desc: throttle point for cap acquisition decay counter
+  desc: The throttle point for cap acquisition decay counter
```

Comment:
Suggest `Threshold at which the cap acquisition decay counter throttles`

Reply:
Done.